aixplain/modules/pipeline/designer/nodes.py

```diff
@@ -129,6 +129,18 @@ def serialize(self) -> dict:
         return obj
 
 
+class BareAssetInputs(Inputs):
+    pass
+
+
+class BareAssetOutputs(Outputs):
+    pass
+
+
+class BareAsset(AssetNode[BareAssetInputs, BareAssetOutputs]):
```

Comment:
What are these classes for?

Reply:
These classes are for concrete(bare) implementation of abstract AssetNode class. AssetNode is an abstract class in mind which expects typed input and output classes.